### PR TITLE
namespaces on assignments were not respected by migrator

### PR DESCRIPTION
**Pull request by beije**
Fixes issue where namespaces on assignments were not respected by the migrator

### CODE REVIEW

File: src/Database/Migration/Assignment/AssignmentMigrator.php [hunk 1]
```diff
@@ -78,20 +78,27 @@ public function migrate(Migration $migration)
         }
 
         foreach ($assignments as $assignment) {
+            $namespace = (isset($assignment['namespace']) ? $assignment['namespace'] : $stream->getNamespace());
```
Comment (Piterden) [inline src/Database/Migration/Assignment/AssignmentMigrator.php]:
```php
$namespace = array_get($assignment, 'namespace', $stream->getNamespace());
```
We are inside Laravel, bro... ))

File: src/Database/Migration/Assignment/AssignmentMigrator.php [hunk 2]
```diff
@@ -78,20 +78,27 @@ public function migrate(Migration $migration)
...
             }
 
             $assignment['field']  = $field;
             $assignment['stream'] = $stream;
 
+            /*
+             * Remove namespace assignment so it's not treated
+             * as a column name in creation step
+             */
+            unset($assignment['namespace']);
```
Comment (Piterden) [inline src/Database/Migration/Assignment/AssignmentMigrator.php]:
```php
array_forget($assignment, 'namespace');
```
It is not necessary, but we are inside Laravel...

**Review by Piterden — CHANGES_REQUESTED** [review]

**Comment by beije** [comment]
@Piterden thanks, I've implemented the changes :)

**Comment by Piterden** [comment]
@RyanThompson congrats about 100th star!! )))

@beije do you have the gitlab account?

**Comment by beije** [comment]
@Piterden No, I don't have a gitlab account(?) :)

**Comment by RyanThompson** [comment]
Nicely done! Thank you.

**Pull request merged**